ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1046,86 +1046,66 @@ def test024_archive_sub_order(self):
 
         self.assertEqual(len(rows_count) - 1, 1)
 
-    # will continue with us
-    @skip('https://modeso.atlassian.net/browse/LIMS-4914')
     def test027_update_material_type(self):
         """
         Apply this on suborder number 5 for example:-
         -When user update the materiel type from table view once I delete it message will appear
         (All analysis created with this order and test plan will be deleted )
         -Once you press on OK button, the material type & article & test pan will delete
         -You can update it by choose another one and choose corresponding article & test plan
+
         LIMS-4264
         """
+        self.info('get random order')
+        orders, _ = self.orders_api.get_all_orders(limit=50)
+        order = random.choice(orders['orders'])
+        suborders, _ = self.orders_api.get_suborder_by_order_id(order['id'])
+        suborder = suborders['orders'][0]
+        suborder_update_index = len(suborders['orders']) - 1
```

Comment:
In this test case you need to update the article in first suborder so you can use sub_order_index = 0 direct and use in when you use update_suborder direct

Reply:
API's index is the reverse of UI index , so I need to revert it to cover multiple suborder case.